conf/bluebear.config

```diff
@@ -1,24 +1,23 @@
+def scratch_dir = "/scratch/${System.getenv("USER")}"
+
 params {
     // Profile config
     config_profile_description = 'BlueBEAR cluster profile'
     config_profile_contact     = 'Alex Lyttle (@alexlyttle)'
     config_profile_url         = 'https://docs.bear.bham.ac.uk/'
-
-    // Maximum resources available
-    max_memory                 = 512.GB
-    max_cpus                   = 112
```

Comment:
These should not be removed otherwise older versions of pipelines will not work!

Reply:
Thanks, good to know. I'll revert it! 

The [tutorial](https://nf-co.re/docs/tutorials/use_nf-core_pipelines/writing_institutional_profiles) in the docs appears to have removed these lines so I assumed they were fully deprecated.

Reply:
Oops, I should add it back in :eyes:

Reply:
Ah it was there, but in a collapsed block: https://github.com/nf-core/website/pull/3186

Reply:
Thanks for fixing on the website 😄 I have reverted the relevant commit.